documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager.Shared/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "contoso-widget-manager-shared",
+  "version": "1.0.0",
+  "description": "",
+  "cadlMain": "main.cadl",
+  "scripts": {
+    "build": ""
+  },
+  "dependencies": {
+    "@azure-tools/cadl-azure-core": "latest"
```

Comment:
This should be reduced to `@azure-tools/cadl-autorest": "latest",`. We also should confirm with the cadl folks whether this should be under dependencies or devDependencies? @allenjzhang does it matter from a cadl perspective?

Reply:
Removed package.json for shared library.